services_and_applications/slate/building_your_own_application/prerequisites.rst

```diff
@@ -0,0 +1,92 @@
+.. _prerequisites:
+
+*******************
+Prerequisites
+*******************
+ 
+The following items need to be established before deploying applications on Slate systems (Marble or Onyx OpenShift clusters).
+
+Establish a project/namespace on Onyx or Marble
+-----------------------------------------------
+
+Follow :ref:`slate_getting_started` if you do not already have a project/namespace established on Onyx or Marble.
+
+Install the OC tool
+-------------------
+
+The OC tool provides CLI access to the OpenShift cluster. It needs to be installed on your machine.
+
+It is a single binary that can be downloaded from a number of places (the choice is yours):
+
+- `Homebrew <https://brew.sh/>`_ on MacOS (need Homebrew setup first): 
+
+ .. code-block:: bash
+
+     brew install openshift-cli 
+
```

Comment:
I am not sure that the homebrew package is always kept up to date so if we have it in the list I think it should be farther down and have a note

```suggestion
- `Homebrew <https://brew.sh/>`_ on MacOS (need Homebrew setup first): 

 .. note::

     The Homebrew package is not always kept up to date with the latest version of OpenShift so some client features may not be available

 .. code-block:: bash

     brew install openshift-cli 

```

Reply:
I did not know you could get directly form the cluster, that is awesome to know. I will probably remove the brew install option, if that is not kept up to date.